### PR TITLE
Fixed swagger files for SDK generation

### PR DESCRIPTION
**Pull request by expedia-rlefrancois**
Added Delete to product reference
Fixed swagger files for SDK generation

### CODE REVIEW

**Comment by exprhartmann** [comment]
Starting review (as I sneakily attend a meeting)

File: images/swagger.json [hunk 1]
```diff
@@ -854,7 +854,7 @@
           "type": "string",
           "description": "Distribution model adopted by the rate plan, matching property configuration.",
           "enum": [
-            "HotelCollect", "ExpediaCollect. ExpediaCollect: indicates that when customers book this rate plan", "they will pay Expedia. HotelCollect: indicates that when customers book this rate plan", "they will pay the property. Note: if a product can be sold as both ExpediaCollect and HotelCollect", "there will be 2 distribution rules under the rate plan to indicate this"
+            "HotelCollect", "ExpediaCollect"
```
Comment (exprhartmann) [inline images/swagger.json]: This had been written by Etienne. Is he cool with this change?

**Comment by exprhartmann** [comment]
Done with review, looks good to me!

File: images/swagger.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/PropertyDTO"
+              "$ref": "#/definitions/ResponseWrapperDTO«List«PropertyDTO»»"
```
Comment (EXPEylazzari) [inline images/swagger.json]: Are we sure about the usage of the `«` and the `»` chars for generics? Is that what the Swagger spec says to use? I would prefer sticking with the Java symbols (e.g. `ResponseWrapperDTO<List<PropertyDTO>>`) or if we can't use those, maybe use the Scala ones (e.g. `ResponseWrapperDTO[List[PropertyDTO]]`).

**Comment by EXPEylazzari** [comment]
Done review. Only one small comment about how generics are expressed.

File: images/swagger.json (hunk 2 above)
Comment (expedia-rlefrancois) [inline images/swagger.json]: That's how spring fox generated it in v2. Its not really a generic for swagger, Its just a DTO with a long name. In the generated SDK, I have a DTO for each variation.

**Pull request merged**